requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+pytest
```

Comment:
dotenv is missing so on my system pytest fails because of it ;)

Reply:
same with playsound (make install installs from requirements-dev.txt and these are missing)

Reply:
sorry will add `-r requirements.txt` to `requirements-dev.txt`, missed that